evap/results/exporters.py

```diff
@@ -169,9 +169,13 @@ def write_headings_and_evaluation_info(
             export_name += f"\n{semesters[0].name}"
         degree_names = [degree.name for degree in Degree.objects.filter(pk__in=degrees)]
         course_type_names = [course_type.name for course_type in CourseType.objects.filter(pk__in=course_types)]
-        self.write_cell(
-            _("{}\n\n{}\n\n{}").format(export_name, ", ".join(degree_names), ", ".join(course_type_names)), "headline"
-        )
+        if verbose_heading:
+            self.write_cell(
```

Comment:
The two paths here are not tested. Testing this should be relatively simple, can you add a small test for the two different headers? (Either at the view level by using two views that use the exporter here, or directly testing the exporter. Not quire sure which makes more sense, it feels like the view level would be a bit more stable)